api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -193,4 +202,21 @@ message RateLimitResponse {
   // - :ref:`envoy.filters.network.ratelimit <config_network_filters_ratelimit_dynamic_metadata>` for network filter.
   // - :ref:`envoy.filters.thrift.rate_limit <config_thrift_filters_rate_limit_dynamic_metadata>` for Thrift filter.
   google.protobuf.Struct dynamic_metadata = 6;
+
+  // Quota is available for a request if its whole descriptor set has cached quota available.
+  // This is a union of all descriptors in the descriptor set. Clients can use the quota for future matches if and only if the descriptor set matches what was sent in the request that originated this response.
+  //
+  // If quota is available, a RLS request will not be made and the quota will be reduced by 1 for all matching descriptor sets.
```

Comment:
```suggestion
  // If quota is available, a RLS request will not be made and the quota will be reduced by 1.
```

(I think this was copied but the last part I don't think makes sense for this case.)

Reply:
done